diff --git a/node_modules/brace-expansion/dist/commonjs/index.js b/node_modules/brace-expansion/dist/commonjs/index.js
index 869a6be..985fc86 100644
--- a/node_modules/brace-expansion/dist/commonjs/index.js
+++ b/node_modules/brace-expansion/dist/commonjs/index.js
@@ -286,4 +286,5 @@ function expand_(str, max, maxLength, isTop) {
     }
     return acc;
 }
+module.exports = Object.assign(expand, exports, { default: expand });
 //# sourceMappingURL=index.js.map
diff --git a/node_modules/brace-expansion/dist/esm/index.js b/node_modules/brace-expansion/dist/esm/index.js
index fd68f57..50a8371 100644
--- a/node_modules/brace-expansion/dist/esm/index.js
+++ b/node_modules/brace-expansion/dist/esm/index.js
@@ -282,4 +282,5 @@ function expand_(str, max, maxLength, isTop) {
     }
     return acc;
 }
+export default expand;
 //# sourceMappingURL=index.js.map
